packages/jest-mock/src/__tests__/index.test.ts

```diff
@@ -1216,6 +1216,18 @@ describe('moduleMocker', () => {
       expect(originalCallArguments[1]).toBe(secondArg);
       expect(spy).not.toHaveBeenCalled();
     });
+
+    it('should not rely on hasOwnProperty', () => {
```

Comment:
The description of the test I find not the best.
It still relies on hasOwnProperty as it is calling it from the Object.prototype
I would name it as `should not throw an error if spying on object without prototype or something`

Reply:
Perhaps it could be simplified to 'should work with object of null prototype'?

Reply:
sounds good

Reply:
Done 👍